ansys/dpf/core/meshes_container.py

```diff
@@ -37,6 +39,63 @@ def __init__(self, meshes_container=None, server=None):
             self, types.meshed_region, collection=meshes_container, server=self._server
         )
 
+    def plot(self, fields_container=None, **kwargs):
+        """Plot the meshes container with a specific result if
+        fields_container is specified.
+
+        Parameters
+        ----------
+        fields_container : FieldsContainer, optional
+            Data to plot. The default is None.
+
+        Examples
+        --------
+        >>> from ansys.dpf import core as dpf
+        >>> from ansys.dpf.core import examples
+        >>> model = dpf.Model(examples.multishells_rst)
+        >>> mesh = model.metadata.meshed_region
+        >>> split_mesh_op = dpf.Operator("split_mesh")
+        >>> split_mesh_op.connect(7, mesh)
+        >>> split_mesh_op.connect(13, "mat")
+        >>> meshes_cont = split_mesh_op.outputs.mesh_controller()
+        >>> disp_op = dpf.Operator("U")
+        >>> disp_op.connect(7, meshes_cont)
+        >>> ds = dpf.DataSources(examples.multishells_rst)
+        >>> disp_op.connect(4, ds)
+        >>> disp_fc = disp_op.outputs.fields_container()
+        >>> meshes_cont.plot(disp_fc)
+
+        """
+        notebook = kwargs.pop("notebook", None)
+        pl = DpfPlotter(notebook=notebook)
+        if fields_container is not None:
+            size = len(fields_container)
+            i = 0
+            while i < size:
+                label_space = fields_container.get_label_space(i)
+                mesh_to_send = self.get_mesh(label_space)
+                if mesh_to_send == None:
+                    raise dpf_errors.DpfValueError(
+                        "Meshes container and result fields "
+                        "container does not have the same scope. "
+                        "Plotting can not be proceeded. "
+                    )
+                field = fields_container[i]
+                pl.add_field(field, mesh_to_send, show_edges=True, **kwargs)
+                i += 1
+        else:
+            import numpy as np
+            import random
+            for mesh in self:
+                r = random.random()
+                g = random.random()
+                b = random.random()
+                pl.add_mesh(mesh,
+                            show_edges=True,
+                            color=np.array([r, g, b, 1]),
+                            **kwargs)
```

Comment:
Allow the user to override `color` arg.
```suggestion
            from random import random
            random_color = "color" not in kwargs
            for mesh in self:
                if random_color:
                    kwargs["color"] = [random(), random(), random()]
                pl.add_mesh(mesh, **kwargs)
```